.travis.yml

```diff
@@ -1,12 +1,27 @@
-language: nodejs
-node_js:
-  - 8
+language: python
+cache: pip
+python:
+  - 2.7
 install:
-  - npm install -g markdownlint-cli --save-dev
-before_script:
-  - markdownlint .
+  - pip install Pygments setuptools docutils mock pillow alabaster commonmark recommonmark mkdocs
```

Comment:
I'd avoid duplication with build.sh. You could make travis use the external script for example.

Reply:
for the moment let's keep it separate, build in travis is not exactly the same as local one (e.g. you don't need to be root)